File: app/src/main/java/com/anenha/weather/app/provider/WeatherServiceCallback.java
package com.anenha.weather.app.provider;


import com.anenha.weather.app.model.Channel;

/**
 * Created by ana.j.saragossa on 20/04/2015.
 */
public interface WeatherServiceCallback {
    void serviceSuccess(Channel channel);
    void serviceFailure(Exception exception);
}

File: app/src/main/java/com/anenha/weather/app/provider/yahooWeather/WeatherServiceCallback.kt
package com.anenha.weather.app.provider.yahooWeather


import com.anenha.weather.app.model.Channel

/**
 * Created by ana.j.saragossa on 20/04/2015.
 */
interface WeatherServiceCallback {
    fun serviceSuccess(channel: Channel)
    fun serviceFailure(exception: Exception)
}
